index.html

```diff
@@ -264,6 +264,57 @@ <h3>
           <p class="arrow2">⌄</p>
         </li>
       </ul>
+      <section class="contactimg">
+        <div class="background-div">
+          <div class="contact-title">
+            <h4>Contact me</h4>
+          </div>
+          <div class="contact-par">
+            <p>
+              If you have an application you are interested in developing, a
+              feature that you need to built or a project that needs coding. I'd
+              love to help with it.
+            </p>
+          </div>
+          <div class="form-container">
+            <form action="https://formspree.io/f/mleanrpl" method="post">
+              <ul class="list-form">
+                <li class="list4">
+                  <input
+                    type="text"
+                    id="name"
+                    maxlength="30"
+                    name="user_name"
+                    placeholder="Name"
+                    required
+                  />
+                </li>
+                <li class="list4">
+                  <input
+                    type="email"
+                    id="mail"
+                    size="30"
+                    name="user_email"
+                    placeholder="email"
+                    required
+                  />
+                </li>
+              </ul>
+              <textarea
+                id="msg"
+                name="user_message"
+                placeholder="Write your message here"
+                maxlength="500"
+                required
+              ></textarea>
+              <button type="submit" class="btn">Get it Touch</button>
+            </form>
```

Comment:
- [ ] Make sure the background of your contact form is as of the given design. As requested by the previous reviewer the `img` tag was supposed to be implemented as a background image looking like decorations according to this [figma](https://user-images.githubusercontent.com/68167810/121414822-c181e380-c96f-11eb-8892-215122b3d8a1.png) template. Please fix this